core/src/main/java/io/questdb/cutlass/pgwire/PGConnectionContext.java

```diff
@@ -1132,6 +1233,8 @@ private void processParse(
             long msgLimit,
             @Transient SqlCompiler compiler,
             @Transient AssociativeCache<Object> factoryCache,
+            @Transient AssociativeCache<Object> statementFactoryCache,
```

Comment:
name of function argument is inconsistent with every other use. This is portalFactoryCache, right?